modules/users/server/config/users.server.config.js

```diff
@@ -11,7 +11,7 @@ var passport = require('passport'),
 /**
  * Module init function
  */
-module.exports = function (app, db) {
+module.exports = function (app) {
```

Comment:
In leaning towards agreeing with 'this can wait' but I wonder what projects built based on master version will do if `db` is now not available for them and they wish to stay up to date...